docs/user/commercial/subscriptions.rst

```diff
@@ -0,0 +1,52 @@
+Read the Docs for Business subscription management
+==================================================
+
+We want to make it easy to manage your billing information.
+You can manage or cancel subscriptions from the :guilabel:`Settings > Subscriptions` page of the :term:`dashboard`.
+
+Managing your subscription
+--------------------------
+
+* Go to https://readthedocs.com/organizations/.
+* Select your organization from this list.
+* Select :guilabel:`Settings` at the top of the page.
+* Select the :guilabel:`Subscription` tab on the left.
+* Click :guilabel:`Manage Subscription`.
```

Comment:
This is what I always send users :+1:

Reply:
Yea, I copied it from the Front autoreply :)